src/loader/bootparam.rs

```diff
@@ -3861,10 +3868,47 @@ impl Default for boot_params {
         unsafe { ::std::mem::zeroed() }
     }
 }
-pub const X86_SUBARCH_PC: _bindgen_ty_1 = 0;
-pub const X86_SUBARCH_LGUEST: _bindgen_ty_1 = 1;
-pub const X86_SUBARCH_XEN: _bindgen_ty_1 = 2;
-pub const X86_SUBARCH_INTEL_MID: _bindgen_ty_1 = 3;
-pub const X86_SUBARCH_CE4100: _bindgen_ty_1 = 4;
-pub const X86_NR_SUBARCHS: _bindgen_ty_1 = 5;
-pub type _bindgen_ty_1 = ::std::os::raw::c_uint;
+pub const x86_hardware_subarch_X86_SUBARCH_PC: x86_hardware_subarch = 0;
+pub const x86_hardware_subarch_X86_SUBARCH_LGUEST: x86_hardware_subarch = 1;
+pub const x86_hardware_subarch_X86_SUBARCH_XEN: x86_hardware_subarch = 2;
+pub const x86_hardware_subarch_X86_SUBARCH_INTEL_MID: x86_hardware_subarch = 3;
+pub const x86_hardware_subarch_X86_SUBARCH_CE4100: x86_hardware_subarch = 4;
+pub const x86_hardware_subarch_X86_NR_SUBARCHS: x86_hardware_subarch = 5;
+#[doc = " enum x86_hardware_subarch - x86 hardware subarchitecture"]
```

Comment:
These doc entries are a bit weird. Any way we can disable them? I don't think they are necessarily needed and they don't look particularly readable either.

Reply:
I was able to get rid of them with a bindgen parameter.